sockeye/inference.py

```diff
@@ -1461,6 +1461,7 @@ def _beam_search(self,
             avoid_states = constrained.AvoidBatch(self.batch_size, self.beam_size,
                                                   avoid_list=raw_avoid_list,
                                                   global_avoid_trie=self.global_avoid_trie)
+            avoid_states.consume(best_word_indices)
```

Comment:
any chance for a test? :)

Reply:
I can't sneak anything by you! Okay, I'll add one.